docs/shared/attributes.adoc

```diff
@@ -18,6 +18,15 @@
 :WFM-RC-NameShort: RainCatcher
 :WFM-RC-Version: 1.0
 :WFM-RC-NodeVersion: 6 LTS
+
```

Comment:
Doesn't seem right that 'KeycloakURL', 'AngularJsURL', etc info is being added as part of Logger Module.

Prob part of another PR (probably https://github.com/feedhenry-raincatcher/raincatcher-docs/pull/36) but made it into here by mistake?

Reply:
global variables but will be used anyway when content is merged.